frontend/packages/dev-console/src/components/topology/componentUtils.ts

```diff
@@ -46,7 +54,10 @@ const canDropEdgeOnNode = (operation: string, edge: Edge, node: Node): boolean =
     return false;
   }
 
-  if (operation === MOVE_CONNECTOR_OPERATION && edge.getTarget() === node) {
+  if (
+    (operation === MOVE_CONNECTOR_OPERATION || operation === MOVE_EV_SRC_CONNECTOR_OPERATION) &&
```

Comment:
Should check `monitor.getItemType()` instead of the `operation` so that we can remove the custom operations.